games-arcade/oshu/oshu-2.0.0.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Lightweight osu! port"
+HOMEPAGE="https://github.com/fmang/oshu"
+SRC_URI="https://www.mg0.fr/oshu/releases/${P}.tar.gz
+	osu-skin? ( https://www.mg0.fr/oshu/skins/osu-v1.tar.gz -> ${PN}-skin-v1.tar.gz )"
+KEYWORDS="~amd64 ~x86"
```

Comment:
Empty line above them.

Reply:
Above which one?
I don’t have much idea between: ``KEYWORDS``, ``SRC_URI``, ``LICENSE`` (I think it’s more likely to be before ``KEYWORDS`` given the position)

Reply:
Between SRC_URI and KEYWORDS.